adp.py

```diff
@@ -66,10 +68,9 @@ def __init__(self):
         csrf_url = parse.urlunparse(
             (self.api_endpoint.scheme, self.api_endpoint.netloc, "csrf", "", "", "")
         )
-        csrf_req = self.websession.get(csrf_url)
+        _ = self.websession.get(csrf_url)
```

Comment:
```python
self.websession.get(csrf_url) # the webui does the same so it might be needed to satisfy the api endpoints
```

Reply:
I wanted to use the fancy "code suggestion" but have not enough github experience :) The assignment here is useless (and leftover from testing) and this is why I suggested only a comment explaining what happens instead because we don't actually need the return-value. The `_`-syntax I usually know from places where an assignment is necessary, e.g. in for-loops

Reply:
Using your suggested version now